packages/framework-types/src/errors/event-handler-global-error.ts

```diff
@@ -1,8 +1,14 @@
 import { GlobalErrorContainer } from './global-error-container'
-import { EventInterface } from '../concepts'
+import { EventInterface, NotificationInterface } from '../concepts'
+import { EventEnvelope } from '../envelope'
 
 export class EventHandlerGlobalError extends GlobalErrorContainer {
-  constructor(readonly eventInstance: EventInterface, originalError: Error) {
+  constructor(
+    readonly eventEnvelope: EventEnvelope | NotificationInterface,
```

Comment:
why `eventEnvelope` can be either `EventEnvelope` or `NotificationInterface` and not one single type?

Reply:
I think @gonzalojaubert could provide a better answer. Still, I believe it's due to distinguishing between a regular event (consumed via the Cosmos change feed processor) and notifications when using the event stream functionality.